src/api/queries/labels.ts

```diff
@@ -0,0 +1,14 @@
+import { gql } from '@apollo/client';
+import { labelData } from '../fragments/label';
+
+/**
+ * Get all the labels that are available and can be assigned to a collection.
+ */
+export const labels = gql`
+  query labels {
+    labels {
+      ...labelData
+    }
+  }
+  ${labelData}
+`;
```

Comment:
We don't really need the query here as the label data is already fetched as part of the `getInitialCollectionFormData` query [here](https://github.com/Pocket/curation-admin-tools/blob/main/src/api/queries/getInitialCollectionFormData.ts) and passed onto the form, and although it will come in handy later when we set up the Labels page, I'd just revert these and all the associated changes here.